### DISCUSSION TITLE
UUID in esp-sync.sh script

### DISCUSSION THREAD

**Discussion by Sithuk** [zbm-dev/zfsbootmenu#365] [General]
The esp-sync.sh script has an ESPC section that has /dev/sdXX partition identifiers. Can UUID identifiers be used instead as /dev/sdXX is not uniquely tied to a specific partition and is prone to change?

https://github.com/zbm-dev/zfsbootmenu/blob/master/contrib/esp-sync.sh

ESPS=(
  "/dev/sdb1"
  "/dev/sdc1"
  "/dev/sdd1"
)

**Comment by ahesford** [comment]
Sure, anything you can pass as the `device` argument to [`mount(8)`](https://man.voidlinux.org/mount.8#Indicating_the_device_and_filesystem) is acceptable.
